designer/server/src/common/components/form-card/_form-card.scss

```diff
@@ -5,6 +5,7 @@
 
 .app-form-card .govuk-button {
   width: 100%;
+  margin-bottom: govuk-spacing(3);
```

Comment:
Can remove this if we add `.govuk-button-group`

```suggestion
```

<br>

Although we'll need to keep full width at tablet and reset the flex direction:

```scss
.app-form-card {
  background-color: govuk-colour("light-grey");
  padding: govuk-spacing(3);
  padding-bottom: 0;

  @include govuk-media-query($from: tablet) {
    .govuk-button-group {
      margin-right: 0;
      flex-direction: column;
      flex-wrap: nowrap;
    }

    .govuk-button {
      width: 100%;
      margin-right: 0;
    }
  }
}
```

Reply:
Good news, I don't think we need this, it looks fine without 🙂  

with the CSS:

```css
.app-form-card {
  background-color: govuk-colour("light-grey");
  padding: govuk-spacing(3);
}

.app-form-card .govuk-button {
  width: 100%;
}

.app-form-card :last-child {
  margin-bottom: 0;
}
```

we get:

mobile
![image](https://github.com/DEFRA/forms-designer/assets/6862563/e9cf645a-a707-4a50-9f5f-262ef9c8ec06)

tablet
![image](https://github.com/DEFRA/forms-designer/assets/6862563/6ca857f5-db05-492f-a614-037469eeda41)

desktop:
![image](https://github.com/DEFRA/forms-designer/assets/6862563/248bbc50-ebba-4979-8bfb-cd4e41f6c463)

Reply:
Not a blocker by any means, just aware that button groups will handles multiple buttons and links together (and their spacing)

Can always add it in future when the delete link goes in